<ui version="4.0" >
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Name:</string>
       </property>
       <property name="buddy" >
        <cstring>nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="nameEdit" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&amp;Address:</string>
       </property>
       <property name="buddy" >
        <cstring>addressEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&amp;Phone:</string>
       </property>
       <property name="buddy" >
        <cstring>phoneEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="phoneEdit" />
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="addressEdit" />
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>phoneEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>257</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>277</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
